arxiv/metadata/metacheck.py

```diff
@@ -195,179 +242,156 @@ def contains_html_elements(s: str) -> bool:
+        report.add_complaint( HOLD, "Title cannot be empty")
     elif len(v) < MIN_TITLE_LEN:
-        disposition = combine_dispositions(disposition, WARN)
-        complaints.append("Title: too short")
+        report.add_complaint( WARN, "Title: too short")
     #
-    if re.match( "title", v, re.IGNORECASE):
-        disposition = combine_dispositions(disposition, WARN)
-        complaints.append("Title: begins with 'title'")
+    if re.match( r"title\b", v, re.IGNORECASE):
+        report.add_complaint( WARN, "Title: begins with 'title'")
     #
     # TODO: leading, trailing, excess space?
     if re.search( r"\\\\", v):
-        disposition = combine_dispositions(disposition, WARN)
-        complaints.append("Title: contains TeX line break")
+        report.add_complaint( WARN, "Title: contains line break")
     #
     if looks_like_all_caps(v):
-        disposition = combine_dispositions(disposition, WARN)
-        complaints.append("Title: excessive capitalization")
+        report.add_complaint( WARN, "Title: excessive capitalization")
     elif long_word_caps(v):
-        disposition = combine_dispositions(disposition, WARN)
-        complaints.append("Title: excessive capitalization (words)")
+        report.add_complaint( WARN, "Title: excessive capitalization (words)")
     #
     if smart_starts_with_lowercase_check(v):
-        disposition = combine_dispositions(disposition, WARN)
-        complaints.append("Title: starts with a lower case letter")
+        report.add_complaint( WARN, "Title: starts with a lower case letter")
     #
     if re.search( r"\\href{", v):
-        disposition = combine_dispositions(disposition, WARN)
-        complaints.append("Title: contains TeX \\href")
+        report.add_complaint( WARN, "Title: contains \\href")
     #
     if re.search( r"\\url{", v):
-        disposition = combine_dispositions(disposition, WARN)
-        complaints.append("Title: contains TeX \\url")
+        report.add_complaint( WARN, "Title: contains \\url")
     #
     if contains_outside_math( r"\\emph", v):
-        disposition = combine_dispositions(disposition, WARN)
-        complaints.append("Title: contains \\emph")
+        report.add_complaint( WARN, "Title: contains \\emph")
     #
     if contains_outside_math( r"\\uline", v):
-        disposition = combine_dispositions(disposition, WARN)
-        complaints.append("Title: contains \\uline")
+        report.add_complaint( WARN, "Title: contains \\uline")
     #
     if contains_outside_math( r"\\textbf", v):
-        disposition = combine_dispositions(disposition, WARN)
-        complaints.append("Title: contains \\textbf")
+        report.add_complaint( WARN, "Title: contains \\textbf")
     #
     if contains_outside_math( r"\\texttt", v):
-        disposition = combine_dispositions(disposition, WARN)
-        complaints.append("Title: contains \\texttt")
+        report.add_complaint( WARN, "Title: contains \\texttt")

```

Comment:
All the `title contains \textXX` - are they actually bad? I used `\emph` etc at times.

Reply:
I implemented the requirements I was given.

We get more cases of \emph in the abstracts; it's not clear what we want to support but we don't support all of TeX, just mathjax.